library/camellia.c

```diff
@@ -560,6 +578,12 @@ int mbedtls_camellia_crypt_cbc( mbedtls_camellia_context *ctx,
 {
```

Comment:
Indentation of parameters above is slightly out.

Reply:
Fixing.